app/views/DescribeTheIdenticalGoodsView.scala.html

```diff
@@ -0,0 +1,73 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import viewmodels.InputWidth._
+@import components._
+
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    govukInput: GovukInput,
+    govukButton: GovukButton,
+    govukTextarea: GovukTextarea,
+    list: BulletList,
+    paragraph: Paragraph,
+    caption: Caption,
+    heading: Heading,
+    cancelLink: CancelApplicationLink
+)
+
+@(form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages("describeTheIdenticalGoods.title"))) {
+
+    @formHelper(action = routes.DescribeTheIdenticalGoodsController.onSubmit(mode)) {
+
+    @caption(messages("describeTheIdenticalGoods.caption"))
+
+    @heading(messages("describeTheIdenticalGoods.heading"))
+
+    @paragraph(Html(messages("describeTheIdenticalGoods.paragraph")))
+
+            @list(
+            Seq(
+                Html(messages("describeTheIdenticalGoods.bulletPoint.1")),
+                Html(messages("describeTheIdenticalGoods.bulletPoint.2")),
+                Html(messages("describeTheIdenticalGoods.bulletPoint.3"))
+            ),
+        "govuk-list govuk-list--bullet",
+)
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form))
+        }
+
+        @govukTextarea(
```

Comment:
Errors have recently moved above the caption:

```suggestion
    @if(form.errors.nonEmpty) {
        @govukErrorSummary(ErrorSummaryViewModel(form))
    }
        
    @caption(messages("describeTheIdenticalGoods.caption"))

    @heading(messages("describeTheIdenticalGoods.heading"))

    @paragraph(Html(messages("describeTheIdenticalGoods.paragraph")))

    @list(
        Seq(
            Html(messages("describeTheIdenticalGoods.bulletPoint.1")),
            Html(messages("describeTheIdenticalGoods.bulletPoint.2")),
            Html(messages("describeTheIdenticalGoods.bulletPoint.3"))
        ),
        "govuk-list govuk-list--bullet",
    )

    @govukTextarea(
```